controllers/useraccount/useraccount_controller_test.go

```diff
@@ -1377,11 +1377,10 @@ func TestReconcile(t *testing.T) {
 		// User is expected to be created in first reconcile
 		updatedUser := assertUser(t, r, userAcc)
 		// Ensure that the user ID has been encoded
-		require.Equal(t, "rhd:b64:QUJDREVGR0hJSktMTU5PUFFSU1RVVldYWVphYmNkZWZnaGlqa2xtbm9wcXJzdHV2d3h5ejAxMjM0"+
-			"NTY3ODlBQkNERUZHSElKS0xNTk9QUVJTVFVWV1hZWmFiY2RlZmdoaWprbG1ub3BxcnN0dXZ3eHl6MDEyMzQ1Njc4OUFCQ0RFRkdISUpLTE"+
-			"1OT1BRUlNUVVZXWFlaYWJjZGVmZ2hpamtsbW5vcHFyc3R1dnd4eXowMTIzNDU2Nzg5QUJDREVGR0hJSktMTU5PUFFSU1RVVldYWVphYmNk"+
-			"ZWZnaGlqa2xtbm9wcXJzdHV2d3h5ejAxMjM0NTY3ODlBQkNERUZHSElKS0xNTk9QUVJTVFVWV1hZWmFiY2RlZmdoaWprbG1ub3BxcnN0dX"+
-			"Z3eHl6MDEyMzQ1Njc4OQ", updatedUser.Identities[0])
+		require.Equal(t, "rhd:ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789"+
```

Comment:
So, it means OpenShift do not support long User IDs, right?
Let's change the test (or even delete it since we do not treat long IDs in any special way anymore?).

Reply:
done

Reply:
(as in removed it)